src/templates/home-page.js

```diff
@@ -185,15 +185,17 @@ export const query = graphql`
     allMarkdownRemark(
       limit: 3,
       sort: {order: DESC, fields: [frontmatter___date]},
-      filter: {frontmatter: {path: {ne: null}, template: {eq: "news-item"}}}
+      filter: {frontmatter: {template: {eq: "news-item"}}}
```

Comment:
How hard would it be to have this also filter on a tag?

Reply:
I don't think it should be too difficult. What do you want to filter?

Reply:
“highlight” or “featured” maybe? Thinking this lets us have a little more control over *which* news items are featured on the front page.

Reply:
Ah, yes! Good idea!